src/sentry/integrations/github/client.py

```diff
@@ -105,11 +105,11 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> JSONData:
 
     def get_trees_for_org(
         self, org_slug: str, gh_org: str, cache_seconds: int = 3600 * 24
-    ) -> JSONData:
+    ) -> Dict[str, RepoTree]:
```

Comment:
This is now more explicit. This forces me to update the tests to instantiate these as well.